biolink-model.yaml

```diff
@@ -6549,6 +6555,170 @@ slots:
+    description: >-
+      The expected (calculated) number of instances in a dataset/cohort whose records contain both the subject and 
+      object concept of an association if the subject and object concepts are independent.
+
+  relative frequency subject:
+    is_a: association slot
+    description: >-
+      The frequency at which subject and object concepts co-occur in EHR
+      records within a dataset/cohort, relative to the frequency at which the subject
+      concept appears in these same records.
+    range: float
+    examples:
+      - value: 0.01840490798
+
+  relative frequency object:
+    is_a: association slot
+    description: >-
+      The frequency at which subject and object concepts co-occur in EHR
+      records within a dataset/cohort, relative to the frequency at which the object
+      concept appears in these same records.
+    range: string
+
+  relative frequency subject confidence interval:
+    is_a: association slot
+    description: >-
+      The 99% confidence interval for the relative_frequency_subject calculation
+      (i.e. the range of values within which the true value has a 99% chance of falling)
+    range: string
+
+  relative frequency object confidence interval:
+    is_a: association slot
+    description: >-
+      The 99% confidence interval for the relative_frequency_object calculation
+      (i.e. the range of values within which the true value has a 99% chance of falling)
+    range: string
+
+  adjusted p value:
+    is_a: association slot
+    description: >-
+      The adjusted p-value is the probability of obtaining test results
+      at least as extreme as the results actually observed, under the assumption that
+      the null hypothesis is correct, adjusted for multiple comparisons.  
+      P is always italicized and capitalized. The actual P value* should be expressed (P=. 04) 
+      rather than expressing a statement of inequality (P<. 05), unless P<.
+    range: float
+
+  bonferonni adjusted p value:
+    is_a: association slot
+    description: >-
+      The Bonferroni correction is an adjustment made to P values when several dependent or independent 
+      statistical tests are being performed simultaneously on a single data set. To perform a Bonferroni 
+      correction, divide the critical P value (α) by the number of comparisons being made.  P is always italicized and 
+      capitalized. The actual P value* should be expressed (P=. 04) rather than expressing a statement of inequality 
+      (P<. 05), unless P<.
+    range: float
+    examples:
+      - value: 0.018
+
+  supporting text:
+    is_a: association slot
+    description: The segment of text from a document that supports the mined assertion.
+    range: string
+    examples:
+      - value: The administration of 50g/ml bupivacaine promoted maximum breast cancer.
+
+  supporting document:
+    is_a: association slot
+    description: The document that contains the supporting text of a Text Mining Result.
```

Comment:
I would generalize the definition here as well - as we will need a slot like this in other contexts.  In our draft EPC model we have proposed a 'has supporting document' property with the following definition: "A publication or other document that reports the statement expressed in an Association, or provides information used as evidence supporting this statement.".  

If Bill / Mike agree this definition applies here (IMO it does), let's use something like this.

Reply:
done